compaction/compaction_manager.cc

```diff
@@ -1037,6 +1030,13 @@ class compaction_manager::offstrategy_compaction_task : public compaction_manage
             }
         }
 
+        // at this moment reshape_candidates contains a set of sstables ready for integration into main set
+        auto desc = sstables::compaction_completion_desc{
+            .old_sstables = std::move(old_sstables),
+            .new_sstables = std::move(reshape_candidates)
+        };
+        co_await t.update_sstable_lists_on_off_strategy_completion(std::move(desc));
```

Comment:
This is shadowing the other desc. It should have a different name, those two descs are different.